packages/KoolTip/KoolTip.vue

```diff
@@ -1,35 +1,45 @@
 <template>
-  <div
-    :aria-hidden="isHovering ? 'true' : 'false'"
-    :data-message="message"
-    :class="positionClass"
-    :style="{'text-align': alignment}"
-    class="k-tooltip"
-    role="tooltip"
-    @mouseout="isHovering(false)"
-    @mouseover="isHovering(true)">
-    <slot/>
-  </div>
+  <KPop
+    :placement="placement"
+    :popover-classes="`kooltip ${computedClass} ${className}`"
+    trigger="hover"
+    width="auto"
+    hide-caret>
+    <slot />
+    <div
+      slot="content"
+      role="tooltip">
+      <slot
```

Comment:
not sure I understand, but slots are distributed so that the default slot of kooltip is the target.

```vue
<Kooltip label="the tip">
  <Component>
</Kooltip>
```

Then I'm passing in the default slot of Kooltip as the default slot of KPop. KPop also has a content slot so I'm adding a slot in there so that users of kooltip can add in their own content slot. Do you have a diff that illustrates your point?

Reply:
Oh duh, IDK why I didn't realize the default slot was the component. NVM This makes total sense 🤦🏼‍♂️